templates/mongodb-atlas.private-endpoint.template.yaml

```
@@ -0,0 +1,267 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Atlas AWS CloudFormation Quickstart with VPC Peering. (qs-1rkorhefe)
```

Can't reuse template IDs. Replace with `qs-1tq8f0hrt`
```suggestion
Description: MongoDB Atlas AWS CloudFormation Quickstart with VPC Peering. (qs-1tq8f0hrt)
```